src/CrossReferences.jl

```diff
@@ -161,6 +165,26 @@ function docsxref(link::Markdown.Link, code, meta, page, doc)
     end
 end
 
+function resolve_reference(mod::Module, ex::Expr, modules)
+    buildname(xdot::Expr) = isa(xdot.args[1], Symbol) ? (xdot.args[1], xdot.args[2].value) : (buildname(xdot.args[1])..., xdot.args[2].value)
```

Comment:
Could you add brief (internal) docstrings explaining what these two functions do? Just to make maintenance easier in the future.